simpleocr/files.py

```diff
@@ -1,9 +1,9 @@
 import os
 import cv2
-from tesseract_utils import read_boxfile, write_boxfile
+from simpleocr.tesseract_utils import read_boxfile, write_boxfile
 
 IMAGE_EXTENSIONS = ['.png', '.tif', '.jpg', '.jpeg']
-DATA_DIRECTORY = 'data/'
+DATA_DIRECTORY = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
```

Comment:
No, not really. The project I wanted to use this for is paused for various reasons. I'll take a look at this as well for my other packages, I didn't find this when searching for including datafiles! Thanks :)